subsystems/PistonControl.java

```diff
@@ -0,0 +1,22 @@
+package org.frc5587.lib.subsystems;
+
+import edu.wpi.first.wpilibj.DoubleSolenoid;
+
+import static edu.wpi.first.wpilibj.DoubleSolenoid.Value;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+public class PistonControl extends SubsystemBase{
+    private DoubleSolenoid[] doubleSolenoidPistons;
+
+    public PistonControl(int[] pistonIDS) {
+        for(int i = 0; i < pistonIDS.length; i+=2) {
```

Comment:
before this for loop, you should add `doubleSolenoidPistons = new DoubleSolenoid[pistonIDS.length/2];` to initialize the array

Reply:
you should also make the 2d array, so the for loop doesn't have the `+=2`, cause that causes issues elsewhere